guides/common/modules/proc_creating-a-playbook-with-modules-from-project-ansible-collection.adoc

```diff
@@ -19,6 +19,7 @@ The following examples use Ansible vault to manage sensitive data.
 * A system that can reach the {Project} API.
 This is the system that you will execute your playbook against.
 It can be the {ProjectServer} itself, a {SmartProxyServer}, or any other system in your environment.
+If there is no system with direct connection to the {Project} API, an HTTP proxy can be used.
 The following examples use `localhost`.
```

Comment:
The bullet point starts with "system that can reach API" and adding a line about "system that can't reach API" contradicts that. I'm wondering how to best restructure the prerequisite to accommodate the new scenario. Perhaps like this?

```suggestion
* A system that you will execute your playbook against.
The following examples use `localhost`.
+
This system must be able to reach the {Project} API.
You can choose from these options:
** A system that can reach the {Project} API.
It can be the {ProjectServer} itself, a {SmartProxyServer}, or any other system in your environment.
** A system without a direct connection to the {Project} API that uses an HTTP proxy to connect to {Project}.
```

Reply:
Well, using a proxy makes the system able to reach the api. Just not *directly*. But I see what you mean.

Yes, I think that's good!